src/engine/objects/mod.rs

```diff
@@ -177,7 +177,19 @@ where
     GlobalCtx: GlobalContext,
     RequestCtx: RequestContext,
 {
-    pub(crate) fn new(
+    /// Returns a new Node.
+    ///
+    /// # Example
+    /// ```rust
+    /// use std::collections::HashMap;
+    /// use warpgrapher::engine::objects::Node;
+    /// use warpgrapher::engine::value::Value;
+    ///
+    /// let mut props = HashMap::<String, Value>::new();
+    /// props.insert("name".to_string(), "Joe".to_string().into());
+    /// let user_node = Node::<(),()>::new("User".to_string(), props);
+    /// ```
+    pub fn new(
```

Comment:
Does Node::new really need to be made public? Aren't custom resolvers going through the ResolverFacade to create nodes?

Reply:
Yes, that was on oversight.

Reply:
Fixed.